packages/backend/tsconfig.json

```diff
@@ -6,6 +6,6 @@
     "rootDir": "./src",
     "declaration": false
   },
-  "include": ["src/**/*"],
+  "include": ["src/**/*", "seeds"],
```

Comment:
I don't think we need to include `seeds`.

Reply:
removed